### PR TITLE
Extend linter

### PR DESCRIPTION
**Pull request by eanplatter**

### CODE REVIEW

**Comment by datwheat** [comment]
Lint jsx also in the example directory

File: package.json [hunk 1]
```diff
@@ -10,9 +10,10 @@
   "scripts": {
     "postinstall": "node src/postinstall/index.js",
     "prestart": "webpack",
-    "lint": "eslint src/*",
+    "lint": "eslint *.js src/**/*.js example/**/*.js",
```
Comment (FiberJW) [inline package.json]: You should also require linting the jsx in the example folder

**Comment by FiberJW** [comment]
lgtm

**Comment by eanplatter** [comment]
LGTM

**Pull request merged**